lms/djangoapps/teams/static/teams/js/spec/views/topic_teams_spec.js

```diff
@@ -51,15 +48,16 @@ define([
                 teamsView = createTopicTeamsView({
                     teams: TeamSpecHelpers.createMockTeams({
                         results: testTeamData
-                    })
+                    }),
+                    showTeams: false,
                 });
             var footerEl = teamsView.$('.teams-paging-footer');
             expect(teamsView.$('.teams-paging-header').text()).toMatch('Showing 1-5 out of 6 total');
             expect(footerEl.text()).toMatch('1\\s+out of\\s+\/\\s+2'); // eslint-disable-line no-useless-escape
             expect(footerEl).not.toHaveClass('hidden');
 
             TeamSpecHelpers.verifyCards(teamsView, testTeamData);
-            verifyActions(teamsView);
+            verifyActions(teamsView, { showOptions: false });
```

Comment:
should this be `showActions`?

Reply:
yes, and derp.  I need to look at these tests in general if my understanding of `this.collection` was wrong also.